arc/src/arc_curie_pme.c

```diff
@@ -33,12 +34,13 @@ void curie_pme_configure(u16_t global_context,
 }
 
 // clear all commits in the network and make it ready to learn
-void curie_pme_forget(void) { reg_write16(FORGET_NCOUNT, 0); }
+void curie_pme_forget(void)
```

Comment:
All for expanding this out.

I think the (void) is a good idea in .h files but I don't know that it matters in the definition. Although maybe it does if there *is* no declaration, so maybe it's a good practice.